pkg/acceptance/helpers/stage_client.go

```diff
@@ -44,9 +44,9 @@ func (c *StageClient) CreateStageWithURL(t *testing.T, id sdk.SchemaObjectIdenti
 	return stage, c.DropStageFunc(t, id)
 }
 
-func (c *StageClient) CreateStageWithDirectory(t *testing.T) (*sdk.Stage, func()) {
+func (c *StageClient) CreateStageWithDirectory(t *testing.T, schemaId sdk.DatabaseObjectIdentifier) (*sdk.Stage, func()) {
```

Comment:
If you want to pass a schema, use the `CreateStageInSchema` (or introduce new CreateStageWithDirectoryInSchema). The current usage is inconsistent with other existing helpers. It's also visible by the usage, we should not need to pass the default test schema.

Reply:
Done.